src/anyio/_backends/_asyncio.py

```diff
@@ -2125,6 +2131,8 @@ async def task_wrapper(scope: CancelScope) -> T_Retval:
             scope._tasks.add(task)
             try:
                 return await func(*args)
+            except CancelledError as exc:
+                raise concurrent.futures.CancelledError(str(exc)) from None
```

Comment:
Won't this break the chain of any exceptions attached to the `CancelledError`? I guess that's probably rare.

Reply:
I'm not sure if the chain is preserved anyway, but I can check if you like.

Reply:
I checked and it doesn't seem to matter. There's a C level black box between that reraise and the calling synchronous code.